### PR TITLE
[PTX-2813] Added app definitions for Cassandra and  MongoDB

### PR DESCRIPTION
**Pull request by vinayakshnd**

### CODE REVIEW

File: drivers/scheduler/k8s/specs/cassandra/cassandra-stress.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cassandra-stress
+  labels:
+    app: cassandra-stress
+spec:
+  replicas: 3
```
Comment (mborodin-px) [inline drivers/scheduler/k8s/specs/cassandra/cassandra-stress.yaml:8]: why do we need 3 replicas for cassandra-stress? I think it should be one instance of cassandra-stress which will run some load using cassandra service endpoint

**Review by mborodin-px — COMMENTED** [review]
please test that cassandra-stress pod should be in CrashLoopBackOff when cassandra service is not available

File: drivers/scheduler/k8s/specs/cassandra/cassandra-stress.yaml (hunk 1 above)
Comment (vinayakshnd) [reply drivers/scheduler/k8s/specs/cassandra/cassandra-stress.yaml:39]: addressed these comments in #556

**Comment by vinayakshnd** [comment]
Raised another PR for this #556 .

**Pull request closed without merging**